prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -125,17 +127,27 @@ else
   if [ $BEAMTYPE == "PbPb" ]; then
     [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
   elif [ $BEAMTYPE == "pp" ]; then
-    ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
-    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    if [ -z $ISPILOTDATA ]; then # configuration of ITS tracking for pilot beam data
+      ITS_CONFIG_KEY+="ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2;ITSCATrackerParam.useDiamond=true;ITSCATrackerParam.diamondPos[2]=35.;ITSCATrackerParam.pvRes=10."
+      [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode sync_misaligned"
+    else
+      ITS_CONFIG_KEY+="fastMultConfig.cutMultClusLow=1;fastMultConfig.cutMultClusHigh=2000;fastMultConfig.cutMultVtxHigh=500;ITSVertexerParam.phiCut=0.5;ITSVertexerParam.clusterContributorsCut=3;ITSVertexerParam.tanLambdaCut=0.2"
+      [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    fi
   elif [ $BEAMTYPE == "cosmic" ]; then
     [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode cosmics"
   else
-    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode async"
+    [ -z ${ITS_CONFIG+x} ] && ITS_CONFIG=" --tracking-mode sync_misaligned"
   fi
 fi
 
 has_processing_step ENTROPY_ENCODER && has_detector_ctf TPC && GPU_OUTPUT+=",compressed-clusters-ctf"
 
+if [ -z $DISABLE_ROOT_OUTPUT ]; then
+  # enable only if root output is written, because it slows down the processing
+  GPU_OUTPUT+=",send-clusters-per-sector"
```

Comment:
actually, I would set `send-clusters-per-sector` by default

Reply:
Ok for me. But what about the "slow-down" hinted at by the comment?

Reply:
For pp this slowdown will be insignificant (and we anyway not going to write routinely root trees), in general, for PbPb after some number of events per TF the trees written w/o this option become unreadable.

Reply:
Ok, then I will change it.

Reply:
Please do not set this by default. It slows down the processing, on the sending and the reading side, and it doubles (temporarily) the SHM that is used, since the data is produced in a linear buffer during clusterization, and then it is copied to per-sector buffers for the ROOT output. And actually, also all consumers will become slower, since they will read the per-sector buffers and then copy it back to a linear buffer.
E.g. it would probably break the full system test due to the slow down.
I think it should only be enabled when the ROOT_OUTPUT is enabled.

Reply:
Ah, this is my fault, I thought it is relevant for writing only. But then, for large PbPb datasets we will not be able to store clusters in the root file, since they exceed the 1GB limit.

Reply:
This change is also in https://github.com/AliceO2Group/AliceO2/pull/7479 which doesn't pass the CI unfortunately. Let's see what can be merged first :)

Reply:
Not done as discussed here.